src/networks/types.ts

```diff
@@ -13,6 +13,7 @@ export enum Network {
   bsc = 'bsc',
   zora = 'zora',
   gnosis = 'gnosis',
+  zkSync = 'zkSync',
```

Comment:
In some places I see network key as lowercase `zksync` and sometimes it's `zkSync`. Should we make all the same?